store/postgres/src/deployment.rs

```diff
@@ -40,6 +40,7 @@ use crate::{block_range::BLOCK_RANGE_COLUMN, primary::Site};
 use graph::constraint_violation;
 
 #[derive(DbEnum, Debug, Clone, Copy)]
+#[PgType = "text"]
```

Comment:
Does leaving the `DbEnum` there automatically convert the `text` to `enum SubgraphHealth`? If so, nice!

Reply:
Yep. I find this approach much nicer than the alternative where we'd do it ourselves like [this](https://github.com/graphprotocol/graph-node/pull/5125/files#diff-4f0442fb210dcfd363ebb2af35c9b706b76d1663cf35adb945ae49054c85f2e9), which I think is much more verbose and error prone.